server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/measure/MeasureToMeasureDto.java

```diff
@@ -60,12 +60,8 @@ public MeasureDto toMeasureDto(Measure measure, Metric metric, Component compone
     return out;
   }
 
-  private static void setVariations(MeasureDto measureDto, MeasureVariations variations) {
-    measureDto.setVariation(1, variations.hasVariation1() ? variations.getVariation1() : null);
-    measureDto.setVariation(2, variations.hasVariation2() ? variations.getVariation2() : null);
-    measureDto.setVariation(3, variations.hasVariation3() ? variations.getVariation3() : null);
-    measureDto.setVariation(4, variations.hasVariation4() ? variations.getVariation4() : null);
-    measureDto.setVariation(5, variations.hasVariation5() ? variations.getVariation5() : null);
+  private static void setVariation(MeasureDto measureDto, MeasureVariations variations) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "MeasureVariations"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)